ONSdigital/design-system
# Update babel preset-env and other dependancies

### What is the context of this PR?

Fixes: #3174

This updates `@babel/preset-env` to remove the caniuse-lite warning. It also updates the lock file so there will be other dependencies within that that are updated unless they are pinned to a specific version but this will be limited minor updates within the rules set in the `package.json`.

I have also deleted my local lock file and re run `yarn` to check for any warnings when recreating that, there were a few so I have fixed as many as I could and now there are only a few that can only be fixed by new releases of dependancies which we will have to wait for.

The node version has also been updated and there are some minor formatting/documentation fixes.

Some VR test images needed updating. These seem to only be minor changes. The main one seems to be the lower case "g" is now a open g on links/buttons and not a closed one. But this actually means its more consistent with the non link font as this uses an open g. This may need to be discussed with Joe and Dina to see if it is a problem. It will have come from one of the minor updates that will have been made within the lock file.

### How to review this PR

- Run `yarn start` on main, see that the warning is printed in the terminal
- Run `yarn start` on this branch, see that the warning is now gone
- Delete your local yarn.lock file and run `yarn` on main, see the number of warnings
- Delete your local yarn.lock file and run `yarn` on this branch, see there are fewer warnings

### Checklist

This needs to be completed by the person raising the PR.
